Formula/feedgnuplot.rb

```diff
@@ -0,0 +1,16 @@
+class Feedgnuplot < Formula
+  desc "Tool to plot realtime and stored data from the command-line."
+  homepage "https://github.com/dkogan/feedgnuplot"
+  url "https://github.com/dkogan/feedgnuplot/archive/v1.38.tar.gz"
+  sha256 "8470129f4de9a663702035f15f098766268044bd0052f2a4de5d662990c72d03"
+
+  def install
+    system "perl", "Makefile.PL", "prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/feedgnuplot", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Otherwise, at least `assert_match version.to_s, shell_output("#{bin}/feedgnuplot --version")`.